app/config/paths.js

```diff
@@ -15,10 +14,14 @@ let confDir = homeDir;
 const devDir = resolve(__dirname, '../..');
 const devConfig = resolve(devDir, conf);
 const defaultConfig = resolve(__dirname, defaultConf);
-const pathLegacy = resolve(homeDir, legacyConf);
 
 const icon = resolve(__dirname, '../static/icon.png');
 
+const keymapPath = resolve(__dirname, '../keymaps');
+const darwinKeys = resolve(keymapPath, 'darwin.json');
```

Comment:
Should't these be `path.join`?
http://tips.tutorialhorizon.com/2017/05/01/path-join-vs-path-resolve-in-node-js/

don't know how resolve vs join is performance/resource-wise, 
it's probably so tiny that it's non-existent 😄  

🤔